charts/kubefed/README.md

```diff
@@ -96,6 +96,7 @@ Delete all kubefed CRDs:
 
 ```bash
 $ kubectl delete crd $(kubectl get crd | grep -E 'kubefed.k8s.io' | awk '{print $1}')
+$ kubectl delete crd $(kubectl get crd | grep -E 'clusterregistry.k8s.io' | awk '{print $1}')
```

Comment:
This line is no longer necessary, as the cluster registry is no longer installed by the chart.  Please revert.

Reply:
Done.